### DISCUSSION TITLE
Secondary status displayed next to username

### DISCUSSION THREAD

**Discussion by wait-what** [stoatchat/discussions#36] [Feature Suggestions]
# Read this!

This is from the maintainer, just popping in to say this discussion should not be used to discuss #9, thank you.

From the author, imagine editing my posts? CRINGE.

---

_This is a replacement for #9_

_I feel like this would allow people to express themselves better, without restricting the feature to a tiny subset of all the information that exists in the universe._

## ^ I retract the point above (scroll down) 

Just imagine the possibilities! People would be able to put small pieces of information in the secondary status, such as their pronouns, their bedtime, their mood, irl name, or anything they'd ever want to make known to everyone who talks to them, instantly.

I feel like this would be a great addition. With a 20-or-so character limit and gray-ish text, users could be able to display small pieces of information next to their username in the message list. 

As a twist that I am not so sure about, it could possibly be different for every server and changing it could be done in the nickname changing section. What do you think?

**Comment by sussycatgirl** [comment]
I agree with this. There's no point in having an info field that can't display info that actually matters tbh

**Comment by wait-what** [comment]
Due to further considerations, I believe this should not replace #9.

A dedicated pronoun field can be useful in more places than to simply show it next to the username. It can be used in other places in the UI, as well as in integrations. That would allow for better UX.

**Reply by Aspie96** [reply]
I still think it should replace #9 in what info to show together with the username.

There is no reason to expect most people would chose their pronouns as the most relevant thing.

**Reply by wait-what** [reply]
Perhaps it doesn't have to be shown next to the username, there  could be a toggle. I just don't believe anymore that these features interfere with each other.

**Comment by wait-what** [comment]
Adding to my original concept

I have seen people on Discord using nicknames to show information about themselves - for example:

- [AFK] Person
- Person | sleeping

A secondary status next to the username would let people express themselves in this exact way, except without having to abuse the nickname feature.

**Reply by phoebehfharris** [reply]
What's wrong with the nickname feature?

**Reply by Rexogamer** [reply]
I think they're referring to the idea of using nicknames exclusively for, well, names, and this feature for stuff like "oh btw I'm asleep"

**Reply by phoebehfharris** [reply]
I know, I just don't particularly see the need to use nicknames exclusively for names.

**Comment by Aspie96** [comment]
I actually support this.

This also means that people can put whatever they think is most important for others to know about them.

If for someone that is pronouns, that's fine. That wouldn't be pronouns for most people, probably, and that is also fine.

**Comment by MikePadge** [comment]
+1 for sanity
